README.md

```diff
@@ -83,49 +87,65 @@ Import all decks in document/decks/ directory, be careful, it still imports if d
 
 ### Tasks 
 
-#### build
+#### Build
 
 ```sh
 $ npm run build
 ```
 Builds the entire application to `dist/`. See `package.json` to set environment variables.
 
-#### server-prod
+#### Server-prod
```

Comment:
Why change first letter to capital: Style consistency. Many of the headers in this document began with capital letters, some did not. Corrected for uniformity.